content/chainguard/chainguard-registry/pull-through-guides/artifactory/artifactory-packages-pull-through/index.md

```diff
@@ -80,12 +80,20 @@ export ARTIFACTORY_HOSTNAME=my-artifactory-hostname
 
 If you aren't sure of these values, you can find them in the command from the **Set Up An Alpine Client** window where you found the token:
 
-```
+```shell
 sudo sh -c "echo 'https://linky:<TOKEN>@linkysorg.jfrog.io/artifactory/cg-wolfi/<BRANCH>/<REPOSITORY>'" >> /etc/apk/repositories
 ```
 
 In this example, the Artifactory user profile is `linky` and the hostname is `linkysorg`.
 
+Note that if your Artifactory user profile is an email address, this command will result in an error unless you percent-encode the `@` sign, like this:
+
+```shell
+sudo sh -c "echo 'https://linky%40example.com:<TOKEN>@linkysorg.jfrog.io/artifactory/cg-wolfi/<BRANCH>/<REPOSITORY>'" >> /etc/apk/repositories
```

Comment:
do we generally use "linkysorg" ? would be better to use example in my opinion

Reply:
We don't have any specific rules in place about this. I just realized that I made a mistake in this update and will push another PR shortly. As part of it I will update the `linkysorg` example to say `example-hostname`.